x-pack/plugin/deprecation/src/test/java/org/elasticsearch/xpack/deprecation/IndexDeprecationChecksTests.java

```diff
@@ -110,6 +112,43 @@ public void testTooManyFieldsCheck() throws IOException {
         assertEquals(0, withDefaultFieldIssues.size());
     }
 
+    public void testChainedMultiFields() throws IOException {
+        XContentBuilder xContent = XContentFactory.jsonBuilder().startObject()
```

Comment:
I'd like this test case to include at least one field that has a non-chained multi-field to verify that the warning message only contains the fields with a problem.

Reply:
👍